lib/models/LifecycleConfiguration.ts

```diff
@@ -89,11 +88,24 @@ export type LifecycleConfigurationMetadata = {
     rules: Rule[],
 };
 
+export type XMLRule = {
```

Comment:
To differentiate between the XMLRule for replication, shall we use a more specific name?

```suggestion
export type XMLLifecycleRule = {
```


... and change the replication rule